README.md

```diff
@@ -9,55 +9,54 @@
 __orbitdeterminator__ is a package written in python3 for determining orbit of satellites based on positional data. Various filtering and determination algorithms are available for satellite operators to choose from.  
 
 ### Installation Instructions
-Run the following commands to install orbitdeterminator:
 
+##### Virtual environment setup
+For this guide we will be using **miniconda**
+**Step 1**: Install miniconda using script and instruction [here](https://docs.conda.io/en/latest/miniconda.html)
+**Step 2**: After installation, create a virtualenv with python3.5 as
```

Comment:
Small problem: all steps are appearing on same line

Reply:
Sorry I was busy these last two days. I did checked the code on online markdown editor before pushing, I'll fix and push a new commit